index.js

```diff
@@ -180,34 +182,15 @@ class DockerExecutor extends Executor {
                     '/bin/sh',
                     '-c',
                     [
-                        // Run the launcher in the background
-                        '/opt/sd/launch',
-                        '--api-uri',
-                        this.ecosystem.api,
-                        '--store-uri',
-                        this.ecosystem.store,
-                        '--emitter',
-                        '/opt/sd/emitter',
-                        config.buildId,
-                        '&',
-                        // Run the logservice in the background
-                        '/opt/sd/logservice',
-                        '--emitter',
-                        '/opt/sd/emitter',
-                        '--api-uri',
+                        // Run the wrapper script
+                        '/opt/sd/run.sh',
+                        `"${config.token}"`,
                         this.ecosystem.api,
-                        '--store-uri',
                         this.ecosystem.store,
-                        '--build',
-                        config.buildId,
-                        '&',
-                        // Wait for both background jobs to complete
-                        'wait $(jobs -p)'
+                        DEFAULT_BUILD_TIMEOUT,
```

Comment:
`executor-docker` is not implemented custom buildtimeout yet. In this PR, pass `DEFAULT_BUILD_TIMEOUT` to the wrapper script as buildtimeout value. This value should be exchanged after buildtimeout is implemented.